packages/apps/demo/src/components/ThemeSwitcher.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2023 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Base, context, Row, Button } from '@lightningjs/ui-components';
+import { Router } from '@lightningjs/sdk';
+
+const themes = {};
+
+export default class ThemeSwitcher extends Base {
+  static __componentName() {
+    return 'ThemeSwitcher';
+  }
+
+  static _template() {
+    return {
+      alpha: 0.001,
+      Background: {
+        rtt: true,
+        rect: true,
+        color: 0xfff000000,
+        w: app.stage.w / app.stage.getRenderPrecision(),
+        h: app.stage.h / app.stage.getRenderPrecision()
+      },
+      Selector: {
+        type: Row,
+        neverScroll: true,
+        autoResizeWidth: true,
+        mount: 0.5,
+        y: app.stage.h / app.stage.getRenderPrecision() / 2,
+        x: app.stage.w / app.stage.getRenderPrecision() / 2,
+        items: ['Base'].map(theme => ({
+          type: Button,
+          title: theme,
+          onEnter() {
+            context
+              .setTheme(themes[theme.toLocaleLowerCase()] || {})
+              .then(() => {
+                Router.focusPage();
+              });
+          }
+        }))
+      }
+    };
+  }
+
+  _handleKey() {
+    return true;
+  }
+
+  _getFocused() {
+    return this.tag('Selector');
```

Comment:
since the goal is for people to use this as a reference app (and this extends the Base component), can we add the static tags array and show how we don't need to use the string name here?

Reply:
Updated